app/javascript/components/set-ownership-form/index.jsx

```diff
@@ -0,0 +1,117 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Grid } from 'patternfly-react';
+import MiqFormRenderer from '../../forms/data-driven-form';
+import { API } from '../../http_api';
+import createSchema from './ownership-form.schema';
+
+class SetOwnershipForm extends Component {
+  state = {
+    initialValues: {},
+    userOptions: [],
+    groupOptions: [],
+  };
+
+  componentDidMount() {
+    const { ownershipItems } = this.props;
+    this.loadInitialData(ownershipItems);
+  }
+
+  /**
+   * Adds neutral options: "Don't Change" and "No Owner"/"No User Group" to the select box options.
+   *
+   * @param ownershipIds Array of element ids being edited in the form. If more than one add "Don't Change" option.
+   * @param type The type of the options to which elements are currently being added. ('user' or 'group')
+   * @param data The data already in the select box. To this the function prepends the new fields.
+   * @param tenantGroupId If the type is 'group' and the ownershipIds.length is one then the value of the 'No User Group' field will be this value.
+   * @returns {*[]}
+   */
+  addOptions = (ownershipIds, type, data, tenantGroupId = '') => {
```

Comment:
Please change `ownershipIds` from ArrayOf(Array) to ArrayOf(Object)
i.e.
`[['some_id', 'some label'],...]` to be something like: `[{id: 'some_id',  label: 'some label'},...]`